.github/workflows/deploy-spaces.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         python3 -m build -w
         aws s3 cp dist/gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl s3://gradio-builds/${{ github.sha  }}/
     - name: Install Hub Client Library
-      run: pip install huggingface-hub==0.8.1
```

Comment:
The internal api of the hub changed in a way that is incompatible with this version of the client library